bootstrap/config/config.go

```diff
@@ -577,36 +611,20 @@ func (cp *Processor) loadConfigYamlFromFile(yamlFile string) (map[string]any, er
 	return data, nil
 }
 
-func (cp *Processor) mergeMapWithConfig(config any, configMap map[string]any) error {
-	// convert the common config passed in to a map[string]any
-	var destConfigMap map[string]any
-	if err := utils.ConvertToMap(config, &destConfigMap); err != nil {
-		return fmt.Errorf("failed to mergre configuration: %v", err)
-	}
-
-	utils.MergeMaps(destConfigMap, configMap)
-
-	if err := utils.ConvertFromMap(configMap, config); err != nil {
-		return fmt.Errorf("failed to mergre configuration: %v", err)
-	}
-
-	return nil
-}
-
 // GetConfigFileLocation uses the environment variables and flags to determine the location of the configuration
 func GetConfigFileLocation(lc logger.LoggingClient, flags flags.Common) string {
 	configDir := environment.GetConfigDir(lc, flags.ConfigDirectory())
 	profileDir := environment.GetProfileDir(lc, flags.Profile())
 	configFileName := environment.GetConfigFileName(lc, flags.ConfigFileName())
 
-	return configDir + pathSep + profileDir + configFileName
+	return configDir + utils.PathSep + profileDir + configFileName
```

Comment:
should this use a filepathpath.Join? that way it is os independent? also does profileDir already have a trailing PathSep?

Reply:
Nope, because this is a Consul path which is always Linux style.

Reply:
Yes, Good catch!

Reply:
reworked